source/NVDAObjects/window/edit.py

```diff
@@ -842,6 +843,19 @@ def updateSelection(self):
 		self.obj.ITextSelectionObject.start=self._rangeObj.start
 		self.obj.ITextSelectionObject.end=self._rangeObj.end
 
+	def getTextInfoForCodepointMovement(self) -> Self:
+		# In PoEdit ITextDocumentTextInfo sometimes cannot access the last character when that character is

```

Comment:
This is also reproducible in Wordpad, so seems to be ITextRange related in general. Could you please update this comment accordingly?